src/addressResolvers/ens.ts

```diff
@@ -63,7 +63,10 @@ export async function resolveNames(handles: Handle[]): Promise<Record<Handle, Ad
     } = await graphQlCall(
       'https://api.thegraph.com/subgraphs/name/ensdomains/ens',
       `query Domains {
-        domains(where: {name_in: ["${normalizedHandles.join('","')}"]}) {
+        domains(where: {
+          name_in: ["${normalizedHandles.join('","')}"],
+          expiryDate_gt: "${Math.floor(Date.now() / 1e3)}"}
```

Comment:
`expiryDate` is null if name is a subdomain example `1.snapspace.eth` should be fine.?

Reply:
Will add some tests for that

Reply:
mhhh... subdomain does not return any address, even without the expiryDate filter.

Should we just convert all subdomains to domains, and use the domain's resolved address ? (and map it back to the subdomain)

Reply:
mmm getting complicated 😢 may be better to return the domain if doesn't have expiryDate for now

Reply:
Issue is that subdomain does not return any result, even on master.